### PR TITLE
chore: GH: auto comment unsigned PR

### PR DESCRIPTION
**Pull request by ddevsr**
**Description**
I think this help
See https://github.com/1Password/check-signed-commits-action

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: .github/workflows/label-signing.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  issues: write
+  pull-requests: write
+
+jobs:
+  build:
+    name: Check Signed Commit
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```
Comment (MGatner) [inline .github/workflows/label-signing.yml:23]:
```suggestion
        uses: actions/checkout@v4
```

**Review by MGatner — CHANGES_REQUESTED** [review]
Checkout v4 is going to take some getting used to 😉

This looks like a great addition.

**Review by MGatner — APPROVED** [review]
LGTM!

File: .github/workflows/label-signing.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      - 'develop'
+      - '4.*'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  issues: write
```
Comment (kenjis) [inline .github/workflows/label-signing.yml:14]: Why do you need `issues: write`?

Comment (ddevsr) [reply .github/workflows/label-signing.yml]: No need, already remove

**Review by kenjis — APPROVED** [review]

**Comment by kenjis** [comment]
```
Run 1Password/check-signed-commits-action@v1
Run unsigned_commits="$(curl -s -H "Authorization: token $GITHUB_TOKEN" "$COMMITS_URL" | jq '.[] | select(.commit.verification.verified == false) | .commit.message')"
Found unsigned commits:
"Skip hostname checks if CURLRequest options 'verify' is set to false.\n\nWhen CURLRequest options 'verify' is set to false, some CURLOPT_SSL_... options should be disabled in such a way as to allow requests to pass through in case the destination is for example on private networks."
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   743  100   153  100   590    444   1714 --:--:-- --:--:-- --:--:--  2159
{
  "message": "Resource not accessible by integration",
  "documentation_url": "https://docs.github.com/rest/issues/comments#create-an-issue-comment"
}
Error: Process completed with exit code 1.
```
https://github.com/codeigniter4/CodeIgniter4/actions/runs/6993466495/job/19026745521

**Comment by kenjis** [comment]
To post comments, it seems `pull_request_target` is needed:
https://github.com/1Password/check-signed-commits-action#pull_request_target-vs-pull_request

**Pull request merged**